python/pyspark/sql/functions.py

```diff
@@ -409,7 +432,7 @@ def isnan(col):
 
 @since(1.6)
 def isnull(col):
-    """An expression that returns true iff the column is null.
+    """An expression that returns true if the column is null.
```

Comment:
`the column`? This is misleading. We should make the Python documents consistent with what we did in Scala. 
For example, `isNull` in Scala APIs is described as
> Returns true if `expr` is null, or false otherwise.

Ref: https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/nullExpressions.scala#L280